packages/rancher-prom2teams/generated-changes/overlay/templates/psp.yaml

```diff
@@ -26,3 +26,4 @@ spec:
   readOnlyRootFilesystem: false
   volumes:
     - 'configMap'
+    - 'secret'
```

Comment:
Just to clarify, are the additions to provide `secret` volume permissions to sachet and prom2teams a followup from a previous PR that added PSPs to the subcharts? If so, can you link the PR in this comment so we can keep track of that?

I think you need these in order to allow service accounts to be mounted onto any Pod and I think I remember the context, but just wanted to double check. No need for any code changes though.

Reply:
Yes, they are the follow-up for this  PR https://github.com/rancher/charts/pull/1119

Reply:
And that PR went into 2.5.8 right ? Otherwise we wouldn't need to bump package version

Reply:
Looking at the "new charts in 2.6" issue, 0.2.0 is already new for 2.6 ? Same for other charts here